source/cursorManager.py

```diff
@@ -149,6 +149,11 @@ def doFindText(self,text,reverse=False,caseSensitive=False):
 			wx.CallAfter(gui.messageBox,_('text "%s" not found')%text,_("Find Error"),wx.OK|wx.ICON_ERROR)
 		CursorManager._lastFindText=text
 		CursorManager._lastCaseSensitivity=caseSensitive
+		try:
+			CursorManager._searchHistory.remove(text)
+		except ValueError:
+			pass
+		CursorManager._searchHistory.insert(0, text)

```

Comment:
Hi,

Consider the following scenario: user goes through several rounds of searching with different texts, and now the user types a previously searched term somehow. What would you do if this happens - just prepend whatever the text is, or see if the term is already part of search history?

Thanks.